server/src/core/server/services/notifications/externalService.ts

```diff
@@ -211,6 +235,84 @@ export class ExternalNotificationsService {
     return false;
   }
 
+  public async createApprove(input: CreateApproveParams) {
+    if (!this.active()) {
+      return false;
+    }
+
+    try {
+      const data = {
+        source: NotificationSource,
+        type: NotificationType.CoralReply,
+        to: this.userToExternalProfile(input.to),
+        story: this.storyToInput(input.story),
+        site: this.siteToInput(input.site),
+        comment: this.commentToInput(input.comment, input.story),
+      };
+
+      return await this.send(data);
+    } catch (err) {
+      this.logger.warn(
+        { err, input },
+        "an error occurred while sending a reply notification"
+      );
+    }
+
+    return false;
+  }
+
+  public async createReject(input: CreateRejectParams) {
+    if (!this.active()) {
+      return false;
+    }
+
+    try {
+      const data = {
+        source: NotificationSource,
+        type: NotificationType.CoralReply,
```

Comment:
I think this type should be something like `CoralCommentRejected`? (and error logging below updated)